src/components/DisplayContainer/DisplayContainer.scss

```diff
@@ -0,0 +1,6 @@
+.display-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding: 10px;
+}
```

Comment:
I love how you laid this out with space-around since different rows will have a different number of cards being displayed.   The cards lay out really nicely.  Once we are displaying the true data we can get a better feel for how big the cards need to be.